src/i_winmusic.c

```diff
@@ -1476,34 +1507,22 @@ static void I_WIN_PlaySong(void *handle, boolean looping)
 
 static void I_WIN_PauseSong(void)
 {
-    MMRESULT mmr;
-
     if (!hMidiStream)
     {
         return;
     }
 
-    mmr = midiStreamPause(hMidiStream);
-    if (mmr != MMSYSERR_NOERROR)
-    {
-        MidiError("midiStreamPause", mmr);
-    }
+    player_state = STATE_STOPPED;
```

Comment:
Why does a function with "Pause" in its name set state to "stopped"?

Reply:
https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/b6ab88a65d9604518213526440dbfb21377f462f